### ISSUE TITLE
Fails to run with shellcheck error

### ISSUE THREAD

**Issue by Chaz6** [bash-lsp/bash-language-server#414]
When I try use this in neovim, it fails with the following output:-

```[START][2022-05-29 11:46:18] LSP logging initiated
[ERROR][2022-05-29 11:46:18] .../vim/lsp/rpc.lua:420    "rpc"   "bash-language-server"  "stderr"        "/home/chaz/.local/share/nvim/lsp_servers/bashls/node_modules/bash-language-server/out/linter.js:98\n                throw new Error(`shellcheck: json parse failed with error ${e}\\nout:\\n${out}\\nerr:\\n${err}`);\n
      ^\n\nError: shellcheck: json parse failed with error SyntaxError: Unexpected end of JSON input\nout:\n\nerr:\nunrecognized option `--source-path=/home/chaz'\n\nUsage: shellcheck [OPTIONS...] FILES...\n  -a                --check-sourced          Include warnings from sourced files\n  -C[WHEN]          --color[=WHEN]           Use color (auto, always, never)\n  -e CODE1,CODE2..  --exclude=CODE1,CODE2..  Exclude types of warnings\n  -f FORMAT         --format=FORMAT          Output format (checkstyle, gcc, json, tty)\n  -s SHELLNAME      --shell=SHELLNAME        Specify dialect (sh, bash, dash, ksh)\n  -S SEVERITY       --severity=SEVERITY      Minimum severity of errors to consider (error, warning, info, style)\n  -V                --version                Print version information\n  -W NUM            --wiki-link-count=NUM    The number of wiki links to show, when applicable.\n  -x                --external-sources       Allow 'source' outside of FILES\n\n\n    at Linter.<anonymous> (/home/chaz/.local/share/nvim/lsp_servers/bashls/node_modules/bash-language-server/out/linter.js:98:23)\n    at Generator.next (<anonymous>)\n    at fulfilled (/home/chaz/.local/share/nvim/lsp_servers/bashls/node_modules/bash-language-server/out/linter.js:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n"
```

**Comment by Chaz6** [comment]
Output of `shellcheck --version`:-
```ShellCheck - shell script analysis tool
version: 0.6.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

**Comment by Chaz6** [comment]
Fixed by upgradeing to 0.8.0. I thought I was on the latest version as I installed it with `stack` but it seems that has an old version.
